docs/rules/aws_resource_tags.md

```diff
@@ -0,0 +1,47 @@
+# aws_resource_tags
+
+Rule for resources tag presence and value validation from prefixed list.
+
+## Example
+
+```hcl
+rule "aws_resource_tags" {
+    enabled  = true
+    exclude  = ["aws_autoscaling_group"]
+    required = ["Environment"]
+    values   = {
+        Department  = ["finance", "hr", "payments", "engineering"]
+        Environment = ["sandbox", "staging", "production"]
+    }
+}
+
+provider "aws" {
+    ...
+    default_tags {
+        tags = { Environment = "sandbox" }
+    }
+}
+
+resource "aws_s3_bucket" "bucket" {
+    ...
+    tags = { Project: "homepage", Department: "science" }
+}
+```
+
+```
+$ tflint
+1 issue(s) found:
+
+Notice: aws_s3_bucket.bucket Received 'science' for tag 'Department', expected one of 'finance,hr,payments,engineering'.
+
+  on test.tf line 3:
+   3:   tags = { Project: "homepage", Department = "science" }
+```
+
+## Why
+
+Enforce standard tag values across all resources.
```

Comment:
```suggestion
Organizations commonly use conventional AWS tags to track resources, such as `Environment`, `Department`, or `Service`. Example AWS features that depend on having a well-known set of tags include:

* [Cost allocation](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/cost-alloc-tags.html)
* [ABAC](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction_attribute-based-access-control.html)
```

Make sure _Why_ tells the user why they should care about enforcing this and doesn't just restate the rule description.